### DISCUSSION TITLE
How to change default dark/light theme colors?

### DISCUSSION THREAD

**Discussion by krivahtoo** [emilk/egui#2050] [Q&A] [answered]
Is there a way to easilly change dafault dark/light theme colors globally. background color, foreground color, etc...

**Comment by ThePerfectComputer** [comment, marked as answer]
You can do ``ctx.set_visuals(egui::Visuals::dark());``

**Reply by krivahtoo** [reply]
@ThePerfectComputer Thanks for the reply. I want to change colors in both dark and light modes. Is that possible with ``ctx.set_visuals(..)``?

**Reply by ThePerfectComputer** [reply]
Oh - I see. You're asking how to change the actual dark/light themes themselves?
So create your own dark and like functions like the ones shown here: https://github.com/emilk/egui/blob/master/crates/egui/src/style.rs#L677

And then modify the values to suit your needs.

**Reply by John-Nagle** [reply]
Any suggestions for good color themes? I'm getting this hard to read "shades of black" layout by default.

![eguishadesofblack](https://user-images.githubusercontent.com/942608/198932913-288eabde-518b-45af-b543-94dcef0ee322.png)

**Comment by White-Rabbit-Scientific** [comment]
Try Catppuccin/egui on Github.
